CONTRIBUTING.md

```diff
@@ -29,12 +29,15 @@ the proposed types, just pick a general issue with no template.
 
 ## Submitting Pull Requests
 
-We take outside code contributions to UMF through GitHub pull requests.
+We take outside code contributions to UMF through GitHub Pull Requests.
```

Comment:
nit: `outside` sounds weird, what do you think about `external code contributions ...`?

Reply:
heh, actually, we take all contributions via PR 😄  updated